keyservice/keyservice.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+option go_package = "/keyservice";
```

Comment:
Added this ad best practice and because I was getting errors because of this as well when doing `make generate`